src/StorageWriter/ClaimController.ts

```diff
@@ -56,7 +56,7 @@ export class ClaimController {
     return { claim }
   }
 
-  private readonly uploadClaim = (claim: SignedVerifiableClaim) => this.ipfs.addText(JSON.stringify(claim))
+  private readonly uploadClaim = (claim: SignedVerifiableClaim) => this.ipfs.addText()(JSON.stringify(claim))
```

Comment:
Tried to use anycPipe like so, but the types are complaining, might be something with the asyncPipe types that need to be adjusted unless I am missing something obvious.

```ts
private readonly configureUploadText = ({ timeout = minutesToMiliseconds(10) } = {}) => this.ipfs.addText({ timeout })

  private readonly uploadClaim = (claim: SignedVerifiableClaim) => asyncPipe(
    JSON.stringify,
    this.configureUploadText()
  )

  private readonly storeClaim = async (data: StoreNextClaimData): Promise<StoreNextClaimData> => {
    const { claim } = data
    const ipfsFileHash = await this.uploadClaim(claim)
    return {
      ...data,
      ipfsFileHash,
    }
  }
```